example/Assets/Standard Assets/Bugsnag/Bugsnag.cs

```diff
@@ -72,7 +73,12 @@ public class NativeBugsnag {
             AndroidJavaObject activity = unityPlayerClass.GetStatic<AndroidJavaObject>("currentActivity");
             AndroidJavaObject app = activity.Call<AndroidJavaObject>("getApplicationContext");
 
-            Bugsnag.CallStatic<AndroidJavaObject> ("init", app, apiKey);
+            jvalue[] args =  new jvalue[2] {
+                new jvalue() { l = app.GetRawObject() },
+                new jvalue() { l = AndroidJNI.NewStringUTF(apiKey) },
+            };
+            IntPtr methodId = AndroidJNI.GetStaticMethodID(BugsnagUnity.GetRawClass(), "init", "(Landroid/content/Context;Ljava/lang/String;)V");
+            AndroidJNI.CallStaticVoidMethod(BugsnagUnity.GetRawClass(), methodId, args);
```

Comment:
Why are we using the JNI directly rather than going through the unity wrapper for it?